### ISSUE TITLE
[bug] preview_width is `0` in `preview.sh`, unexpected

### ISSUE THREAD

**Issue by alexzanderr** [kamiyaa/joshuto#187]
Hello.
Im trying to make the preview work well, by using this method:
```shell
        rs|md|toml)
            echo "hello: ${PREVIEW_WIDTH}"
            bat "${FILE_PATH}" \
                --terminal-width="${PREVIEW_WIDTH}" \
                --wrap=auto \
                --color=always \
                --pager=never \
                --style=plain \
                --number \
                && exit 5
            ;;
```
But when I'm running the `joshuto` bin and trying to preview a readme for example:
![image](https://user-images.githubusercontent.com/54772464/180601966-c2ff1ca4-af89-4bc8-9489-8300fb084b73.png)
the preview_width is zero, not good.

But there is another problem, sometimes the above issue happens, sometimes doesnt happen and `preview_width` is ok and then I can preview:
![image](https://user-images.githubusercontent.com/54772464/180602430-5a8df0be-5856-4f66-a855-d87da8b858f9.png)

What is happening?

**Comment by kamiyaa** [comment]
Apologies that I never got around to this, are you still able to reproduce this?

**Comment by kamiyaa** [comment]
Should be fixed with 1908ff94b7e5f981a4254ca697056a299b2e6b51

Let me know if its fixed!

**Comment by alexzanderr** [comment]
hello. sorry for a long time no response!

i can confirm that now its working fine, in the latest commit: 269cf09

thanks for the response and for the bug fix!
